tensorboard/plugins/pr_curve/tf_pr_curve_dashboard/tf-pr-curve-card.html

```diff
@@ -35,15 +34,22 @@
       description="[[tagMetadata.description]]"
     ></tf-card-heading>
 
-    <div id="chart-and-spinner-container">
-      <vz-line-chart
+    <div id="chart-container">
+      <tf-line-chart-data-loader
         x-components-creation-method="[[_xComponentsCreationMethod]]"
         y-value-accessor="[[_yValueAccessor]]"
         tooltip-columns="[[_tooltipColumns]]"
         color-scale="[[_colorScaleFunction]]"
         default-x-range="[[_defaultXRange]]"
         default-y-range="[[_defaultYRange]]"
-      ></vz-line-chart>
+        smoothing-enabled="[[_smoothingEnabled]]"
+        request-manager="[[requestManager]]"
+        runs="[[runs]]"
+        tag="[[tag]]"
+        data-url="[[_dataUrl]]"
+        process-data="[[_createProcessDataFunction()]]"
+        active="[[_active]]"
+      ></tf-line-chart-data-loader>
       <template is="dom-if" if="[[loading]]">
```

Comment:
This spinner should be removed; the correct spinner is managed by the `tf-line-chart-data-loader`.

As a corollary, is it possible to dissolve the `div#chart-container`?

Reply:
Done. And yes! Removed.